netci.groovy

```diff
@@ -14,7 +14,7 @@ class Constants {
                            'OSX10.12':'latest-or-auto',
                            'Ubuntu':'20170118']
 
-    def static scenarios = ['coreclr', 'corefx']
+    def static scenarios = ['coreclr', 'corefx', 'wasm']
```

Comment:
I think it would be more natural to treat `wasm` as an OS, not a test configuration.

So instead of adding it here, maybe adding it to `osList` as `Windows_NT_Wasm` (and adding to `imageVersionMap`, etc.) would end up nicer now, and would set us up to be able to actually test it with CoreCLR/CoreFX tests in the future.

Reply:
Done (using @crummel's changes)